packages/react/src/components/TextArea/TextArea.styles.ts

```diff
@@ -2,6 +2,7 @@ import { css, pxToRem } from '@project44-manifest/react-styles';
 
 export const useStyles = css({
   '.manifest-textfield-base__input': {
+    resize: 'both',
     minHeight: pxToRem(56),
```

Comment:
It may break the UX for existing TextArea usages which are not expecting this.
If you want to have it in manifest, it's better to make it as configured prop - `resize?: 'none' | 'vertical' | 'horizontal' | 'both'` defaults to none. or just override the prop in place where you need that with `styled` function or inline override like this: `<TextArea css={{ textarea: { resize: 'both' } }} />`